DuckDuckGo/Preferences/View/PreferencesAboutView.swift

```diff
@@ -33,52 +33,37 @@ extension Preferences {
 
         var body: some View {
             PreferencePane {
-                GeometryReader { geometry in
```

Comment:
Having the `GeometryReader` causes the problem. Seems specific to Xcode 16, as a previous build with the same implementation doesn't have this issue.